solanches/models.py

```diff
@@ -78,6 +78,16 @@ def add_destaques(comercio_nome, destaques):
         comercio = Comercio.get_by_name(comercio_nome)
         Cardapio.add_destaques(comercio.get("cardapio"), destaques)
 
+    @staticmethod
+    def get_atributos(comercio_nome):
+        comercio = Comercio.get_by_name(comercio_nome)
+        atributos = comercio.get("attributes")
+        return atributos
+
+    @staticmethod
+    def update_by_nome(comercio_nome, attributes):
+        db.comercio.update_one({"nome": comercio_nome}, {"$set": {"attributes": attributes}})
```

Comment:
Acho legal que a edição seja feita a partir do id do comércio e não do nome, só para o caso de haver alguma mudança em relação a como trataremos os nomes dos comércios. Além disso, seria bom refatorar os parâmetros, colocando as querys para edição como variáveis definidas antes e não diretamente no parâmetro da função. Isso facilita bastante a leitura.

```suggestion
        db.comercio.update_one({"nome": comercio_nome}, {"$set": {"attributes": attributes}})
```